WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -183,6 +183,19 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     ///
     func createdWordPressComAccount(username: String, authToken: String) { }
 
+    func shouldDisplayError(_ error: Error) -> Bool {
```

Comment:
This is the implementation of the two new methods in WPAuthenticatorDelegate. First we signal that we want to handle errors (this should be more granular, and return true only for certain error codes)